### PR TITLE
Set up env variables for Email

### LINKED ISSUE #73: Set up Env variables for Emails
**Describe the bug**
Currently, automated emails are sent to students and recruiters when they register. We want to set up env variables so that we can stop this feature if needed in production by just changing the env variables.

**Expected behavior**
Set up env variables in [.env.example](https://github.com/anshulahuja98/spc_portal/blob/master/.env.example) file. There should a check before every email.

### PR DESCRIPTION
**Pull request by MohanChhabaria**
### Description
Added a env variable to avoid automated emails during production

Fixes #73 

### Type of Change:

- Code
- Quality Assurance
- Test

**Code/Quality Assurance Only**
- Bug fix (non-breaking change which fixes an issue)
- New feature (non-breaking change which adds functionality pre-approved by maintainers)

### Checklist:

- [ ] My PR follows the style guidelines of this project
- [ ] Any dependent changes have been merged 

**Code/Quality Assurance Only**
- [ ] My changes generate no new warnings 
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by MohanChhabaria** [comment]
@Tanuj22  i have made a pull request .Can you please check it

**Comment by Tanuj22** [comment]
> @Tanuj22 i have made a pull request .Can you please check it

Yes. checking now

File: src/accounts/models.py [hunk 1]
```diff
@@ -65,7 +65,9 @@ def student_register_email(self):
             message = EmailMultiAlternatives(subject=subject, body=text_content, from_email=from_email, to=to_email,
                                              connection=connection)
             message.attach_alternative(html_content, "text/html")
-            message.send()
+            send_mail = settings.SEND_EMAIL
+            if send_mail is True:
```
Comment (Tanuj22) [inline src/accounts/models.py:69]: The extra variable defined is not necessary. directly use `settings.SEND_MAIL`

File: src/accounts/models.py [hunk 2]
```diff
@@ -125,7 +127,9 @@ def company_register_email(self):
...
```
Comment (Tanuj22) [inline src/accounts/models.py:131]: Same

File: src/accounts/models.py [hunk 3]
```diff
@@ -144,7 +148,9 @@ def company_details_email(self):
...
```
Comment (Tanuj22) [inline src/accounts/models.py:152]: Same

**Comment by MohanChhabaria** [comment]
@Tanuj22  I have made the required changes.

**Comment by Tanuj22** [comment]
@MohanChhabaria I am thinking it would be better if the checks are taking place [here](https://github.com/anshulahuja98/spc_portal/blob/master/src/accounts/views.py#L34)  and [here](https://github.com/anshulahuja98/spc_portal/blob/master/src/accounts/views.py#L72). This will avoid unnecessary function call

**Comment by MohanChhabaria** [comment]
@Tanuj22  I have made the changes as suggested

**Comment by Tanuj22** [comment]
From next time keep better commit messages

**Pull request merged**